src/NuGet.Core/NuGet.CommandLine.XPlat/Utility/MSBuildAPIUtility.cs

```
@@ -288,20 +302,256 @@ private static ProjectItemGroupElement CreateItemGroup(Project project, string f
             }
         }
 
+        public Dictionary<string, Tuple<IEnumerable<PRPackage>, IEnumerable<PRPackage>>> GetPackageReferencesForList(
+            Project project, string projectPath, IList<string> userInputFrameworks, bool transitive)
+        {
+            //Path to the assets file
+            var assetsPath = project.GetPropertyValue(ASSETS_FILE_PATH_TAG);
+
+            //If the property or file were not found, print an error message and return
+            if (assetsPath == "" || !File.Exists(assetsPath))
```

what if this is `packages.config` based project where there wont be any assets file but still you dont want to raise this error. You just simply wants to ignore such projects.

So you need a way to only filter out `PackageReference` based projects and also a simple error message to indicate that `packages.config` based projects are not supported with this command.

---

This checks if the assets file was not found for any reason whether it's because the project is `packages.config` or restore was not run. I do not know if there is a way to say for sure which one it is.

---

We do that by checking `RestoreProjectStyle` and `TargetFramework` properties... By using these properties, you can understand if there will be an assets file or not.

---

`RestoreProjectStyle` does not always have a value actually. I'll look at `TargetFramework` and see if I can figure out how to do this

---

There's a target that evaluates the RestoreProjectStyle. 

The same should be used for consistency sake.